airflow/models/__init__.py

```diff
@@ -2563,6 +2568,25 @@ def pre_execute(self, context):
         """
         pass
 
+    def get_executor_config(self, executor_config):
+        """Try to import base_executor_config and merge supplied
```

Comment:
I think function docs MUST be in same format as in `def execute(self, context):`
e.g.
```
def get_executor_config(self, executor_config):
    """
    Docs starts here
    """
```

Reply:
You mean doc string must look like your example? flake8 did raise any errors

Reply:
yep, try to be consistent with all functions.